core/common/src/main/java/alluxio/network/protocol/RPCProtoMessage.java

```diff
@@ -186,10 +186,12 @@ public static RPCProtoMessage createResponse(Protocol.Status.Code code, String m
       Throwable e, DataBuffer data) {
     Protocol.Status status = Protocol.Status.newBuilder().setCode(code).setMessage(message).build();
     if (e != null) {
-      Protocol.Exception exception =
-          Protocol.Exception.newBuilder().setClassName(e.getClass().getCanonicalName())
-              .setMessage(e.getMessage()).build();
-      status = status.toBuilder().setCause(exception).build();
+      Protocol.Exception.Builder builder = Protocol.Exception.newBuilder();
+      builder.setClassName(e.getClass().getCanonicalName());
```

Comment:
According to the javadoc, it can return null in the case it is an anonymous class. In our usage it might not ever be null.